### ISSUE TITLE
winproxy.py missing in dev branch?

### ISSUE THREAD

**Issue by 0x2Adr1** [hakril/PythonForWindows#5]
Hey,

`winproxy.py` is not present in the dev branch, is that expected? Is it supposed to be generated?

I added some stuff to `kernel32.py`, ran `generate.py` and `setup.py` but since `winproxy.py` is missing I am not able to use my newly added functions.

```
$ python LoadLibrary.py
Traceback (most recent call last):
  File "LoadLibrary.py", line 4, in <module>
    import windows
  File "C:\Python27\lib\site-packages\pythonforwindows-0.4-py2.7.egg\windows\__init__.py", line 19, in <module>
    from windows import winproxy
ImportError: cannot import name winproxy
```

So I edited `setup.py` to include `windows.winproxy` and `windows.winproxy.apis` to the packages list:

```python
setup(
    # [...]
    packages = ['windows',
                'windows.crypto',
                'windows.debug',
                'windows.generated_def',
                'windows.native_exec',
                'windows.rpc',
                'windows.utils',
                'windows.winobject',
                'windows.winproxy',           # <----- Added package
                'windows.winproxy.apis'],     # <----- Added package
    classifiers = ['Programming Language :: Python :: 2 :: Only',
                   'Programming Language :: Python :: 2.7']
)
```

Now it works, but I am not sure if it is the intended way?

Thanks

**Comment by hakril** [comment]
Hi,

Indeed, ``winproxy.py`` have been refactored and replaced by the ``winproxy/`` directory.
As I am a not very meticulous about what I push on dev I forgot to push the updated ``setup.py``.

This should be resolved with the last commit e5f87e14534aa942bbec2b0e0005d8e1e15b5953.

If you need/want me to add some APIs to ``winproxy`` do not hesitate to create an issue :)

Thanks,

**Comment by 0x2Adr1** [comment]
Thanks for the fix! Closing this issue...
